webcompat/templates/issue.html

```diff
@@ -1,135 +1,120 @@
+        </div>
+
+        <div class="wc-IssueDetail-report">
+          <script type="text/template" id="issue-info-tmpl">
+          <div class="wc-Comment">
+            <div class="wc-Comment-avatar">
+              <img src="<%= reporterAvatar %>" alt="<%= reporter %>">
+            </div>
+            <div class="wc-Comment-body wc-Comment-body--caret">
+              <div class="wc-Comment-wrapper">
+                <div class="wc-Comment-header">
+                  <span class="wc-Comment-owner">
+                    <a href="https://github.com/<%= reporter %>"><%= reporter %></a>
+                  </span> reported on <a href="https://webcompat.com/issues/<%= number %>"><%= createdAt %></a>
+                </div>
+                <div class="wc-Comment-content">
+                  <!-- QR Code -->
+                  <div class="wc-Qr-wrapper">
+                    <span class="wc-QrImage-wrapper">
+                      <button class="wc-QrImage-launcher"
+                              aria-pressed="false"
+                              role="button"
+                      >
+                        <span class="wc-Qr-icon wc-Icon wc-Icon--qrcode" aria-hidden="true"></span>
+                        <span class="wc-Qr-iconPressed wc-Icon wc-Icon--times" aria-hidden="true"></span>
+                        <span class="wc-sronly">Show QR code</span>
+                      </button>
+                    </span>
+                  </div>
+                  <div class="wc-IssueDetail-details">
+
+                    <%= body %>
+                  </div>
+                </div>
+              </div>
+            </div>
           </div>
           </script>
         </div>
-      </div>
-      <!-- Issue Labels -->
-      <div class="Label-wrapper">
-        <script type="text/template" id="issue-labels-tmpl">
-          <span class="Label Label--title">Labels</span>
-          {% if session.user_id and session.avatar_url %}
-          <span class="LabelEditor-wrapper">
-            <button class="LabelEditor-launcher wc-Icon wc-Icon--gear" aria-hidden="true">
-              <span class="wc-sronly">Edit Labels</span></button>
-            </span>
-          {% endif %}
-          <span class="Label-list">
-          <% _.each(labels, function(label) { %>
-            <span class="Label Label--badge" style="background-color:#<%=label.color%>">
-              <%= label.name %>
-            </span>
-          <% }); %>
-          </span>
-        </script>
-      </div>
-
-      <div class="wc-IssueDetail-report">
-        <script type="text/template" id="issue-info-tmpl">
-        <div class="wc-Comment">
+        <!-- Issue Comments -->
+        <div class="wc-IssueDetail-comment">
+          <script type="text/template" id="comment-tmpl">
           <div class="wc-Comment-avatar">
-            <img src="<%= reporterAvatar %>" alt="<%= reporter %>">
+            <img src="<%= avatarUrl %>" alt="<%= commenter %>">
```

Comment:
You probably didn't --- I think the diff is just getting confused with line changes (I think you just moved some stuff around). The note was more for me. :)